opendbc/car/honda/carstate.py

```diff
@@ -90,7 +90,7 @@ def __init__(self, CP):
     super().__init__(CP)
     can_define = CANDefine(DBC[CP.carFingerprint][Bus.pt])
     self.gearbox_msg = "GEARBOX"
-    if CP.carFingerprint == CAR.HONDA_ACCORD and CP.transmissionType == TransmissionType.cvt:
+    if CP.carFingerprint == CAR.HONDA_NBOX or (CP.carFingerprint == CAR.HONDA_ACCORD and CP.transmissionType == TransmissionType.cvt):
```

Comment:
```suggestion
    if CP.carFingerprint in (,CAR.HONDA_ACCORD, CAR.HONDA_NBOX) and CP.transmissionType == TransmissionType.cvt):
```

Reply:
Honda n-box has CVT for all grade and Honda accord doesn't it for all grade.
This condition express ↑, but Do I need modify this line to your suggestion?

Reply:
If you are wanting to use the GEARBOX_15T message like the accord then leave it. But all cars should be in an array here. Look how it's done in other parts of this code, shouldn't be an 'or' statement

Reply:
OK. I changed this.